skrub/tests/test_minhash_encoder.py

```diff
@@ -261,3 +261,52 @@ def test_check_fitted_minhash_encoder() -> None:
     # Check that it works after fitting
     encoder.fit(X)
     encoder.transform(X)
+
+
+def test_deterministic():
+    """Test that the encoder is deterministic"""
+    # TODO: add random state to encoder
+    encoder1 = MinHashEncoder(n_components=4)
+    encoder2 = MinHashEncoder(n_components=4)
+    X = np.array(["a", "b", "c", "d", "e", "f", "g", "h"])[:, None]
+    encoded1 = encoder1.fit_transform(X)
+    encoded2 = encoder2.fit_transform(X)
+    assert np.array_equal(encoded1, encoded2)
```

Comment:
Here, let's directly use:
```python
from numpy.testing import assert_array_equal

...

assert_array_equal(encoded_1, encoded_2)
```
Using numpy for asserting is handy because it comes with many flavours, like `assert_array_almost_equal`.